docs/reference/migration/migrate_8_0/_cluster-node-setting-changes.asciidoc

```diff
@@ -0,0 +1,891 @@
+[discrete]
+[[breaking_80_cluster_node_setting_changes]]
+==== Cluster and node setting changes
+
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+//tag::notable-breaking-changes[]
+TIP: {ess-setting-change}
+
+.You can no longer set `xpack.searchable.snapshot.shared_cache.size` on non-frozen nodes. {ess-icon}
```

Comment:
TODO: This is whitelisted as a safe Cloud setting, but I don't think it makes much sense. I need to follow up with the Distributed team before publishing.

Reply:
I reached out to the Distributed team. We can handle this fix as a separate work effort. The solution will likely impact Cloud users so I'll leave the icon for now.